exercises/simple-linked-list/linked_list_test.go

```diff
@@ -165,46 +165,50 @@ func BenchmarkNewList(b *testing.B) {
 }
 
 func BenchmarkListSize(b *testing.B) {
+	list := New(array1To10)
+	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		b.StopTimer()
-		list := New(array1To10)
-		b.StartTimer()
 		_ = list.Size()
 	}
 }
 
 func BenchmarkListPush(b *testing.B) {
 	for i := 0; i < b.N; i++ {
 		b.StopTimer()
-		list := New(array1To10)
+		list := New([]int{})
 		b.StartTimer()
-		list.Push(11)
+		for k := 0; k < 100; k++ {
+			list.Push(k)
+		}
 	}
 }
 
 func BenchmarkListPop(b *testing.B) {
 	for i := 0; i < b.N; i++ {
 		b.StopTimer()
-		list := New(array1To10)
+		list := New([]int{})
+		for k := 0; k < 100; k++ {
```

Comment:
Same situation as for BenchmarkListPush - 100 push / pop operations is not enough. I suggest 1000.